test/examples/uct_hello_world.c

```diff
@@ -1,17 +1,635 @@
 /**
-* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
-*
+* Copyright (C) UT-Battelle, LLC. 2015. ALL RIGHTS RESERVED.
 * See file LICENSE for terms.
 */
 
-#define HAVE_CONFIG_H /* Force using config.h, so test would fail if header
-                         actually tries to use it */
+#include "ucx_hello_world.h"
+
+#include <ucs/type/status.h>
+
+/*
+ * NOTE: async.h includes <valgrind/memcheck.h> w/o NVALGRIND definition,
+ *       it causes compilation error on system w/o installed valgrind
+ */
+#ifndef NVALGRIND
+#define NVALGRIND 1
+#endif /* NVALGRIND */
+#include <ucs/async/async.h>
 #include <uct/api/uct.h>
-#include <stdio.h>
 
+#define CHKERR_JUMP(_cond, _msg, _label)            \
```

Comment:
Could you please use this helpful macro in ucp_hello_world example in appropriate places?